### PR TITLE
upgrade from 3.x latest to 4.x latest

### PR DESCRIPTION
**Pull request by Ameenasuhani**
Description: upgrade from 3.x latest to 4.x latest on rhel 7

Logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1582538368912/

   The error in ceph ansible purge is filed here [1785736](https://bugzilla.redhat.com/show_bug.cgi?id=1785736)

Signed-off-by: ameenasuhani ameenasuhani@gmail.com

### CODE REVIEW

File: ceph/ceph.py [hunk 1]
```diff
@@ -544,10 +544,7 @@ def check_health(self, rhbuild, client=None, timeout=300):
         if not all(state in lines for state in valid_states):
             logger.error("Valid States are not found in the health check")
             return 1
-        if rhbuild.startswith('4'):
```
Comment (XDsubhash) [inline ceph/ceph.py]: ceph -s , output is different for rhceph 3.x and 4.x , those runs will fail if you edit this health function in framwork.Try to parse json output here and make it compatible with all suites or write a separate health check function for upgrade scenarios

**Comment by XDsubhash** [comment]
@Ameenasuhani can you have the suite and config yaml file name changed to upgrade scenario,it would be easy to understand ,create new yamls if needed. for eg: 3.x_GA_to_4.0/4.1_latest_upgrade

**Comment by XDsubhash** [comment]
@Ameenasuhani can you post the upgrade suite logs ran frm your branch?

**Comment by Ameenasuhani** [comment]
logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1583208270845

**Comment by Ameenasuhani** [comment]
> @Ameenasuhani can you have the suite and config yaml file name changed to upgrade scenario,it will be easy to understand ,create new yamls if needed. for eg: 3.x_GA_to_4.0/4.1_latest_upgrade

**Comment by Ameenasuhani** [comment]
@XDsubhash I have changed the suite and config YAML file name as per your comment.

**Pull request merged**